homeassistant/components/rejseplanen/__init__.py

```diff
@@ -1 +1,106 @@
 """The rejseplanen component."""
+
+from __future__ import annotations
+
+import logging
+
+from homeassistant.const import Platform
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import ConfigEntryNotReady
+from homeassistant.helpers import (
+    config_validation as cv,
+    device_registry as dr,
+    issue_registry as ir,
+)
+from homeassistant.helpers.device_registry import DeviceEntryType
+from homeassistant.helpers.typing import ConfigType
+
+from .const import DOMAIN
+from .coordinator import RejseplanenConfigEntry, RejseplanenDataUpdateCoordinator
+
+_LOGGER = logging.getLogger(__name__)
+
+# Set py_rejseplan library to WARNING level by default to reduce log noise
+# Users can override in configuration.yaml with: py_rejseplan: debug
+logging.getLogger("py_rejseplan").setLevel(logging.WARNING)
```

Comment:
what is it normally then?

Reply:
But I believe the log level is warning by default. IMO we shouldn't touch loggers